### PR TITLE
analog. add update related commands

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: docs/analog/commands/UpdateRun.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+### Examples
+
+| Field        | Value | Hex    |
+| ------------ | ----- | ------ |
+| extra flag   | `31`  | `0x1f` |
+| command id   | `44`  | `0x2c` |
+| command size | `0`   | `0x00` |
+
+Message hex dump with LRC: `1f 2c 00 33`
+
```
Comment (DarkPark) [inline docs/analog/commands/UpdateRun.md:32]:
```suggestion


```

File: docs/analog/commands/UpdateRun.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## Response
+
+### Format
+
+| Size         | Type   | Field             |
+| ------------ | ------ | ----------------- |
+| extra flag   | `31`   | `0x1f`            |
```
Comment (DarkPark) [inline docs/analog/commands/UpdateRun.md:39]: wrong format

File: docs/analog/commands/WriteImage.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Message hex dump with LRC: `1f 2a 14 00 00 00 40 00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f 61`
+
+## Response
+
+### Format
+
+| Size       | Type        | Field                                  |
+| ---------- | ----------- | -------------------------------------- |
+| extra flag | `31`        | `0x1f`                                 |
```
Comment (DarkPark) [inline docs/analog/commands/WriteImage.md:51]: wrong format

File: docs/analog/commands/UpdateRun.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+It's a command with a [three-bytes header](../message.md#command-with-a-three-bytes-header).
+
+### Parameters
+
+The command has no parameters.
+
+### Examples
+
+Message hex dump with LRC: `1f 2c 00 33`
```
Comment (DarkPark) [inline docs/analog/commands/UpdateRun.md:51]: would be nice to have a full example body

and in other commands

File: docs/analog/commands/WriteImage.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+### Parameters
+
+#### **offset**
+
+The offset at which to write the image content to the device's flash memory.
+
+#### **status**
+
+`1` - the execution was successful <br>
+`0` - time execution failed
```
Comment (DeadByElpy) [inline docs/analog/commands/WriteImage.md:67]: execution failed by timeout? or just left as "execution failed"

File: docs/analog/commands/WriteImage.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+### Examples
+
+| Field        | Value                                | Hex                                  |
+| ------------ | ------------------------------------ | ------------------------------------ |
+| extra flag   | `31`                                 | `0x1f`                               |
+| command id   | `42`                                 | `0x2a`                               |
+| command size | `20`                                 | `0x14`                               |
+| offset       | `64`                                 | `0x40`                               |
+| offset       | `0x000102030405060708090a0b0c0d0e0f` | `0x000102030405060708090a0b0c0d0e0f` |
```
Comment (DeadByElpy) [inline docs/analog/commands/WriteImage.md:41]: offset -> image content/image

File: docs/analog/commands/UpdateRun.md (hunk 4 above)
Comment (Pnom) [reply docs/analog/commands/UpdateRun.md:51]: Sorry, but that is the message hex dump with LRC. What do you mean by full example body?

File: docs/analog/commands/UpdateRun.md (hunk 1 above)
Comment (Pnom) [reply docs/analog/commands/UpdateRun.md:32]: fixed

File: docs/analog/commands/UpdateRun.md (hunk 2 above)
Comment (Pnom) [reply docs/analog/commands/UpdateRun.md]: fixed

File: docs/analog/commands/VerifyImage.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+Message hex dump with LRC: `1f 2b 00 34`
+
+## Response
+
+### Format
+
+| Size         | Type   | Field             |
+| ------------ | ------ | ----------------- |
+| extra flag   | `31`   | `0x1f`            |
```
[the remark this answers is not in the document]
Comment (Pnom) [reply docs/analog/commands/VerifyImage.md]: fixed

File: docs/analog/commands/WriteImage.md (hunk 3 above)
Comment (Pnom) [reply docs/analog/commands/WriteImage.md]: fixed

File: docs/analog/commands/WriteImage.md (hunk 5 above)
Comment (Pnom) [reply docs/analog/commands/WriteImage.md]: fixed

File: docs/analog/commands/WriteImage.md (hunk 6 above)
Comment (Pnom) [reply docs/analog/commands/WriteImage.md]: fixed

File: docs/analog/commands/UpdateRun.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+## Response
+
+### Format
+
+| Size | Type   | Field               |
+| ---- | ------ | ------------------- |
+| `1`  | `byte` | extra flag = `0x1f` |
+| `1`  | `byte` | command id = `0x2c` |
+| `1`  | `byte` | command size        |
+
```
Comment (DarkPark) [inline docs/analog/commands/UpdateRun.md:43]:
```suggestion
```

**Review by DarkPark — APPROVED** [review]

**Pull request merged**